src/main/java/com/gamingmesh/jobs/Jobs.java

```diff
@@ -58,60 +38,38 @@
 import com.gamingmesh.jobs.api.JobsExpGainEvent;
 import com.gamingmesh.jobs.api.JobsPrePaymentEvent;
 import com.gamingmesh.jobs.commands.JobsCommands;
-import com.gamingmesh.jobs.config.BlockProtectionManager;
-import com.gamingmesh.jobs.config.BossBarManager;
-import com.gamingmesh.jobs.config.ConfigManager;
-import com.gamingmesh.jobs.config.ExploreManager;
-import com.gamingmesh.jobs.config.GeneralConfigManager;
-import com.gamingmesh.jobs.config.LanguageManager;
-import com.gamingmesh.jobs.config.NameTranslatorManager;
-import com.gamingmesh.jobs.config.RestrictedAreaManager;
-import com.gamingmesh.jobs.config.RestrictedBlockManager;
-import com.gamingmesh.jobs.config.ScheduleManager;
-import com.gamingmesh.jobs.config.ShopManager;
-import com.gamingmesh.jobs.config.TitleManager;
-import com.gamingmesh.jobs.config.YmlMaker;
-import com.gamingmesh.jobs.container.ActionInfo;
-import com.gamingmesh.jobs.container.ActionType;
-import com.gamingmesh.jobs.container.ArchivedJobs;
-import com.gamingmesh.jobs.container.BlockProtection;
-import com.gamingmesh.jobs.container.Boost;
-import com.gamingmesh.jobs.container.CurrencyType;
-import com.gamingmesh.jobs.container.DBAction;
-import com.gamingmesh.jobs.container.FastPayment;
-import com.gamingmesh.jobs.container.Job;
-import com.gamingmesh.jobs.container.JobInfo;
-import com.gamingmesh.jobs.container.JobProgression;
-import com.gamingmesh.jobs.container.JobsPlayer;
-import com.gamingmesh.jobs.container.Log;
-import com.gamingmesh.jobs.container.PlayerInfo;
-import com.gamingmesh.jobs.container.PlayerPoints;
-import com.gamingmesh.jobs.container.QuestProgression;
+import com.gamingmesh.jobs.config.*;
+import com.gamingmesh.jobs.container.*;
```

Comment:
Why did you change it to '*'? I know this is more readable and takes up less space, but we have to use the full path of the packages. Usually only a few developers use it.

Reply:
It has changed by Intellij IDEA automatically. It doesn't affect performance as far as I know.